src/directLine.ts

```diff
@@ -411,11 +413,16 @@ export class DirectLine implements IBotConnection {
             if (options.token && options.conversationId) {
                 this.streamUrl = options.streamUrl;
             } else {
-                console.warn('streamUrl was ignored: you need to provide a token and a conversationid');
+                console.warn('DirectLineJS: streamUrl was ignored: you need to provide a token and a conversationid');
             }
         }
+        
+        const interval = Math.min(~~options.pollingInterval, POLLING_INTERVAL_LOWER_BOUND);
 
-        if (options.pollingInterval !== undefined) {
+        if (options.pollingInterval &&
+            Math.min(~~options.pollingInterval, POLLING_INTERVAL_LOWER_BOUND) < POLLING_INTERVAL_LOWER_BOUND) {
```

Comment:
I this supposed to be `interval` instead of `Math.min(~~options.pollingInterval, POLLING_INTERVAL_LOWER_BOUND)`?

Reply:
ah, yeah.